test/function/error.cpp

```diff
@@ -85,6 +85,8 @@ namespace {
 
```

Comment:
Add your new test to this file somewhere. I would start by copying the `TEST( GetErrorId, Basic )` as a starting point and modifying it to test `stumpless_get_error_id_string` with an invalid id.